### PR TITLE
Component updates

### PR DESCRIPTION
**Pull request by flossypurse**
## What does this PR do?

- Changes custom strings arguments into arrays for the RelatedReadList and CenteredImage components.
- Updates use of these components
- Deletes test/unused components

### CODE REVIEW

File: docs/content/what-is-temporal.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ Temporal is a scalable and reliable runtime for Reentrant Processes called Tem
 imagePath="/diagrams/temporal-intro.svg"
 imageSize="50"
 title="The Temporal system"
-legendstring="π?Workflow Execution"
+legend={[
+["π","Workflow Execution"],
```
Comment (sw-yx) [inline docs/content/what-is-temporal.md:22]: are we really sticking to this pi thing

File: docs/components/RelatedReadList.js [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
-    const parts = item.split("?");
-    const tagStuff = tagInfo(parts[2]);
+  for (const item of readlist) {
+    // validate 
+    const text = item[0];
+    if (text.startsWith('/') || text.startsWith('#')) throw new Error(`readlist item "${text}" looks like a URL, did you forget the sequence?`)
+    const goTo = item[1];
+    if (!goTo.startsWith('/') || !goTo.startsWith('#')) throw new Error(`goTo parameter ${goTo} doesnt start with / or #: was there a mistake?`)
+    const tagStuff = tagInfo(item[2]);
+    if (tagStuff instanceof Error) throw tagStuff;
```
Comment (sw-yx) [inline docs/components/RelatedReadList.js]: added some validation... see if you agree with it. havent run the code yet

**Review by sw-yx — APPROVED** [review]
suggested some validation steps as best practice

File: docs/components/RelatedReadList.js [hunk 3]
```diff
@@ -72,9 +85,12 @@ function tagInfo(tagChar) {
       tagClass = "archetype-tag-explanation";
       tag = "explanation";
       break;
-    default:
+    case "r":
       tagClass = "archetype-tag-reference";
       tag = "reference";
+      break;
+    default:
+      return new Error("unrecognized tag class " + tagChar);
```
Comment (sw-yx) [inline docs/components/RelatedReadList.js:93]: having permissive defaults will cause you headaches down the road. use "r" when you mean "r".

File: docs/content/what-is-temporal.md (hunk 1 above)
Comment (flossypurse) [reply docs/content/what-is-temporal.md:22]: Maybe, have to get @dtornow involved there and not something I want to tackle in this particular PR.

File: docs/components/RelatedReadList.js (hunk 2 above)
Comment (flossypurse) [reply docs/components/RelatedReadList.js]: Ya, the component relies on the array argument to have certain items in the proper order.
Not sure how much we validate that. But I understand your point.

File: docs/components/RelatedReadList.js (hunk 3 above)
Comment (flossypurse) [reply docs/components/RelatedReadList.js:93]: Ya, OK I agree. Will update.

**Pull request merged**